style.css

```diff
@@ -1,109 +1,114 @@
 body {
   scroll-behavior: smooth;
-  background-color: rgba(228, 218, 218, 0.36);
   margin: 0;
-  padding: 0%;
-  width: auto;
+  padding: 0;
 }
 
+.visible-area {
+  width: 90%;
+  margin: 0 auto;
+}
+
+/* Top background contain */
+.contain-img1 {
+  display: flex;
+  justify-content: left;
+}
+
+/* Top background photo */
+.background-img1 {
+  margin: 0;
+  left: 0;
+  top: 0;
+}
+
+/* Class for the hamburguer button */
+.option-menu {
+  display: none;
+}
+
+/* Include the main name -->Erik */
 .main-name {
   font-family: 'Playfair Display', serif;
   font-weight: 400px;
   font-size: 60px;
   line-height: 60px;
-  margin: 100px 84px 0 16px;
-  display: flex;
-  flex-wrap: wrap;
+  align-content: left;
 }
 
+/* Include the main text top */
 .main-text {
   font-family: 'Poppins', sans-serif;
-  font-weight: 400px;
+  font-weight: 200px;
   font-size: 16px;
   line-height: 21px;
   letter-spacing: 3px;
-  margin: 25px 16px 0 16px;
 }
 
+/* Button start collaboration */
 .start-collaboration {
   background-color: #1a2236;
   width: 288px;
   height: 50px;
   font-family: 'Poppins', sans-serif;
   font-size: 16px;
-  line-height: 24px;
   margin: 0 auto;
-  margin-top: 40px;
   display: flex;
   justify-content: center;
   align-items: center;
+  margin-top: 40px;
 }
 
+/* The ancor of start collaboration */
 .start-collaboration a {
   text-decoration: none;
   color: #fff;
 }
 
-.background-img1 {
-  position: absolute;
-  margin: 0;
-  padding: 60px;
-  background-image: url(/assets/image_geometry_1.png);
-  background-repeat: no-repeat;
-}
-
+/* Don't show the imagen 2 in the bis screen */
 .background-img2 {
-  margin-top: 40px;
-  height: 240px;
-  width: 480px;
-  background-image: url(/assets/image_geometry_2.png);
-  background-repeat: no-repeat;
-  background-size: 100%;
-}
-
-/* Class for the hamburguer button */
-.option-menu {
-  background-image: url(/assets/ic_menu.png);
-  background-repeat: no-repeat;
-  position: absolute;
-  padding: 10px;
+  display: none;
 }
 
 /* This first edition is a mobile version */
-@media all and (max-width: 770px) {
+@media only screen and (max-width: 770px) {
   body {
-    border: solid;
+    padding: 0;
     width: 100%;
-    padding-bottom: 400px;
+    overflow-x: hidden;
   }
 
-  .visible-area {
-    max-width: 100%;
+  .nav-bar {
+    display: flex;
+    justify-content: right;
   }
 
-  .start-collaboration {
-    margin-bottom: 40px;
+  .option-menu {
+    color: #000;
+    display: block;
+    font-size: 2em;
+    position: fixed;
+    top: 30px;
+    right: 30px;
```

Comment:
- [OPTIONAL] I suggest removing the `border` and `background-color` from the `menu-icon`. to do this you can use:
```
    border: none;
    background: none;
```